terraform/lib/dependabot/terraform/file_parser.rb

```diff
@@ -218,13 +241,43 @@ def source_type(source_string)
 
       def parsed_file(file)
         @parsed_buildfile ||= {}
+        # handle terragrunt's new format
+        if file.name.include? ".hcl"
+          parsed_file_helper(
+            file,
+            "#{terraform_parser_path('hcl2json')} tmp.tf"
+          )
+        end
+        # handle terraform 0.11 and 0.12
+        begin
+          parsed_file_helper(
+            file,
+            "#{terraform_parser_path('json2hcl')} -reverse < tmp.tf"
+          )
+        rescue SharedHelpers::HelperSubprocessFailed => e
+          @msg = "hcl2json: " + e.message.strip + "\n"
+          begin
+            parsed_file_helper(
+              file,
+              # terraform-config-inspect only works when provided a directory
+              "terraform-config-inspect --json .",
+              # specifying $GOPATH/bin:$PATH here doesn't seem to work..
+              "PATH" => "/opt/go/gopath/bin:$PATH"
```

Comment:
I've experienced issues with this when running on osx, as I did not specify GOPATH and thus the modules where at `/Users/username/go/bin` 

I Would suggest to use `terraform_parser_path` here as well and rather modify the helper script to copy the binary to the correct location